packages/video_player/video_player/example/ios/RunnerTests/VideoPlayerTests.m

```diff
@@ -70,4 +70,89 @@ - (void)testDeregistersFromPlayer {
   [self waitForExpectationsWithTimeout:1 handler:nil];
 }
 
+- (void)testVideoControls {
+  NSObject<FlutterPluginRegistry> *registry =
+      (NSObject<FlutterPluginRegistry> *)[[UIApplication sharedApplication] delegate];
+  NSObject<FlutterPluginRegistrar> *registrar =
+      [registry registrarForPlugin:@"TestVideoControls"];
+
+  FLTVideoPlayerPlugin *videoPlayerPlugin =
+    (FLTVideoPlayerPlugin *)[[FLTVideoPlayerPlugin alloc] initWithRegistrar:registrar];
+
+  NSDictionary<NSString *, id> *videoInitialization = [self testPlugin:videoPlayerPlugin uri:@"https://flutter.github.io/assets-for-api-docs/assets/videos/bee.mp4"];
+  XCTAssertEqualObjects(videoInitialization[@"height"], @720);
+  XCTAssertEqualObjects(videoInitialization[@"width"], @1280);
+  XCTAssertEqualWithAccuracy([videoInitialization[@"duration"] intValue], 4000, 200);
+}
+
+- (void)testAudioControls {
+  NSObject<FlutterPluginRegistry> *registry =
+      (NSObject<FlutterPluginRegistry> *)[[UIApplication sharedApplication] delegate];
+  NSObject<FlutterPluginRegistrar> *registrar =
+      [registry registrarForPlugin:@"TestAudioControls"];
+
+  FLTVideoPlayerPlugin *videoPlayerPlugin =
+    (FLTVideoPlayerPlugin *)[[FLTVideoPlayerPlugin alloc] initWithRegistrar:registrar];
+
+  NSDictionary<NSString *, id> *audioInitialization = [self testPlugin:videoPlayerPlugin uri:@"https://cdn.pixabay.com/audio/2021/09/06/audio_bacd4d6020.mp3"];
```

Comment:
Why a separate repo instead of landing the mp3 here and using a raw GitHub URL?

Reply:
Checking binaries into git repos is generally frowned upon since it never gets removed from the git history.  I think `assets-for-api-doc` is the side-repo other `flutter` org repos use to localize the sin to one place.
https://github.com/flutter/plugins/blob/eb6e912329af4aea97de2183f885483ecc605e44/packages/video_player/video_player/example/lib/main.dart#L222
https://github.com/flutter/plugins/blob/7686be7ecbe1cc049423715b5e2d897cb63a43c1/packages/video_player/video_player_web/example/integration_test/video_player_web_test.dart#L27

We can host them somewhere else, but we should host the videos and images all together.

Reply:
https://github.com/flutter/plugins/pull/4720